src/test/java/com/example/schedule/demo/controller/ScheduleControllerTest.java

```diff
@@ -134,5 +135,59 @@ class ScheduleControllerTest {
         verify(scheduleServiceImpl).createTable(form.getTitle(), form.getScheduleDate(), form.getScheduleTime());
 
     }
+
+    @Test
+    void 指定したパスとidで情報が変更できるか() throws Exception {
```

Comment:
細かいですが、〇〇できるか、というテスト名だと正常系か異常系かの判別がつかないので、〇〇できること、と言った形の方がわかりやすいかと思います。
```suggestion
    void 指定したidでデータが変更できること() throws Exception {
```

Reply:
承知しました、以後気をつけます！
修正しました。[コミットハッシュ](https://github.com/aokiayukodesu/Schedule-list/pull/14/commits/4e13f437cb591497aca94033bb6cb7a683a4ee85)